roqoqo/tests/integration/operations/spin_boson_operations.rs

```diff
@@ -179,22 +264,41 @@ fn operate_one_mode_one_spin(op: Operation, op_param: Operation, name: &str) {
 
     // (2) Test hqslang function
     assert_eq!(op.hqslang(), String::from(name));
+}
 
-    // (3) Test is_parametrized function
+#[test_case(
+    Operation::from(QuantumRabi::new(1, 0, "test".into())),
+    "QuantumRabi"
+)]
+#[test_case(
+    Operation::from(LongitudinalCoupling::new(1, 0, "test".into())),
+    "LongitudinalCoupling"
+)]
+#[test_case(
+    Operation::from(JaynesCummings::new(1, 0, "test".into())),
+    "JaynesCummings"
+)]
+fn is_parametrized(op_param: Operation, name: &str) {
     assert!(!op.is_parametrized());
```

Comment:
I believe a test with `is_parametrized_false` would be appropriate for when these operations aren't parametrised, and to test the new ones

Reply:
Added